stl/inc/utility

```diff
@@ -436,11 +436,12 @@ struct tuple_size<const _Tuple> : _Tuple_size_sfinae<_Tuple> { // size of const
 };
 
 template <class _Tuple>
-struct tuple_size<volatile _Tuple> : _Tuple_size_sfinae<_Tuple> { // size of volatile tuple
+_CXX20_DEPRECATE_VOLATILE struct tuple_size<volatile _Tuple> : _Tuple_size_sfinae<_Tuple> { // size of volatile tuple

```

Comment:
Clang test failures are telling us that attributes need to go between `struct` and the name `tuple_size`. (Occurs lots - I always forget this.)

Reply:
> Clang test failures are telling us that attributes need to go between `struct` and the class name. (Occurs lots - I always forget this.)

Reply:
Thanks, I was scratching my head. Luckily search & replace is my second best friend after copy & paste